### ISSUE TITLE
Cornerstone does not find config.yml

### ISSUE THREAD

**Issue by ahorn42** [vizor-games/infraworld-cornerstone#16]
`java -jar .\target\infraworld-cornerstone.jar` tells me to put my `config.yml` to

```
FATAL com.vizor.unreal.config.Config:231 - Please, put your config.yml into C:\Users\username\Desktop\temp\infraworld-tests\infraworld-cornerstone\target
```

even if a file called `config.yml` exists in this path.

Runing on Windows 10 with 
```
openjdk 14.0.1 2020-04-14
OpenJDK Runtime Environment AdoptOpenJDK (build 14.0.1+7)
OpenJDK 64-Bit Server VM AdoptOpenJDK (build 14.0.1+7, mixed mode, sharing)
```

**Comment by ahorn42** [comment]
same with Oracle Java 8
```
java version "1.8.0_251"
Java(TM) SE Runtime Environment (build 1.8.0_251-b08)
Java HotSpot(TM) 64-Bit Server VM (build 25.251-b08, mixed mode)
```

**Comment by ShuyingMaggieMi** [comment]
I met the same problems when I run the java -jar command. Do you solve your problem now?

**Comment by ShuyingMaggieMi** [comment]
Just got the same problem. I even copy the base configuration file (src/main/resources/config.yml) into the same level with infraworld-cornerstone.jar. it does not work.  Do you solve it?

**Comment by ahorn42** [comment]
nope - my work around is to run the `src\main\java\com\vizor\unreal\Main.java` in debug mode (with vs code with java extension) and modifying the config file located at `src\main\resources\config.yml`

**Comment by ShuyingMaggieMi** [comment]
Thanks man! I am installing JDK11 to give it a try :D
I also wonder do you finally get your unreal engine connected to the server with protobuff via the Infraworld Solution?













At 2020-09-02 17:48:44, "ahorn42" <notifications@github.com> wrote:

nope - my work around is to run the src\main\java\com\vizor\unreal\Main.java in debug mode (with vs code with java extension) and modifying the config file located at src\main\resources\config.yml

**Comment by ahorn42** [comment]
yeah, I got it running, but it was a hard and long way, but it runs very nicely if you got it running ;-) - It seems as the example requires another version (most likely a older version) of the plugin as it is on githubs master branch. But I am using the master right now. Unfortunately due to confidentiality I am not allowed to share the project with you :-/

**Comment by ShuyingMaggieMi** [comment]
Thanks. Got it. I am gonna try with my own version.


On 09/07/2020 17:42, ahorn42 wrote:

yeah, I got it running, but it was a hard and long way, but it runs very nicely if you got it running ;-) - It seems as the example requires another version (most likely a older version) of the plugin as it is on githubs master branch. But I am using the master right now. Unfortunately due to confidentiality I am not allowed to share the project with you :-/

**Comment by ShuyingMaggieMi** [comment]
Hi bud,
When I was running the Main.java  (with vs code with java extension), I could not debug. I have JDK 11 and full vs code extension installed.
The report message is like this:







Do I need to add arguments in launch.json file? Or do you know how to debug Main.java successfully?


Best,
Maggie





At 2020-09-02 17:48:44, "ahorn42" <notifications@github.com> wrote:

nope - my work around is to run the src\main\java\com\vizor\unreal\Main.java in debug mode (with vs code with java extension) and modifying the config file located at src\main\resources\config.yml

**Comment by ahorn42** [comment]
Hmpf - as I tried to look up what I exactly did I found out it isn't working for me anymore :-/ (but also your answer/report message is not visible on github, so I can't compare it) - maybe an update broke up things? But I can't remember that I have changed anything to the `launch.json`

I am currently getting a `com/sun/jdi/ClassNotLoadedException` Exception when hitting the Run or Debug button ...

**Comment by ahorn42** [comment]
I am using openjdk 14

```
openjdk 14.0.2 2020-07-14
OpenJDK Runtime Environment AdoptOpenJDK (build 14.0.2+12)
OpenJDK 64-Bit Server VM AdoptOpenJDK (build 14.0.2+12, mixed mode, sharing)
```
